### ISSUE TITLE
[Errno 111] Connection refused

### ISSUE THREAD

**Issue by h-rezaei** [Antergos/Cnchi#384]
Hello
Just downloaded the latest iso from the website. cnchi doesn't work, gives me lots of errors. removed cnchi  with pacman, manually removed /usr/bin/cnchi-dev. installed cnchi-dev, Still cnchi doesn't work and gives me the same error. Error is pasted in this link:
http://slexy.org/view/s20FPQYYOd

**Comment by Wyn10** [comment]
Were you on an internet connection while starting the setup?

**Comment by Mikaela** [comment]
Cnchi doesn't work for me either with working internet connection. The not-dev version said that it cannot format /dev/sda4 to ext4 (to be checked as I never told it to touch that partition which is "Microsoft Reserved" (maybe Cnchi doesn't like Windows 10)) and the dev version threw me that same API key error.

**Comment by Mikaela** [comment]
Booting the iso finds another issue, https://github.com/Antergos/antergos-iso/issues/33.

* ![](https://vps.mikaela.info/~mikaela/2015-08-25-cnchi/20150825_003.jpg)
    * Partitioning, the scrolling down just say that there isn't free space left.
* ![](https://vps.mikaela.info/~mikaela/2015-08-25-cnchi/20150825_004.jpg)
    * Confirmation on partitioning
* ![](https://vps.mikaela.info/~mikaela/2015-08-25-cnchi/20150825_005.jpg)
    * Actually start the installing and *Error: coudn't format partition `/dev/sda3` with identifier `ext4`* which makes no sense to me.
* ![](https://vps.mikaela.info/~mikaela/2015-08-25-cnchi/20150825_006.jpg)
    * but why would our cnchi care about any formatting error of the new root?
* ![](https://vps.mikaela.info/~mikaela/2015-08-25-cnchi/20150825_007.jpg)
    * which could possibly be guessed.

**Comment by h-rezaei** [comment]
My internet connection is fine, I can ping and download with pacman.
There's something to note though: I cannot connect to antergos's main 
repos. I have to  edit the antergos-mirrorlist. Still, When i do that, I 
can't start cnchi.

**Comment by karasu** [comment]
Just so you know, the API error is harmless, so something else is happening here...

**Comment by karasu** [comment]
@Mikaela : By any chance could you still test Cnchi against that setup again? Latest version is 0.13.40

**Comment by Mikaela** [comment]
> By any chance could you still test Cnchi against that setup again? Latest version is 0.13.40

No, sorry, I moved to vanilla Arch Linux which didn't have any install issues.

**Comment by karasu** [comment]
@Mikaela and @h-rezaei : Thank you both for your reports, but I'm afraid we're unable to reproduce your errors here. Without any more info I'm afraid we won't be able to fix this.

**Comment by h-rezaei** [comment]
Sorry @Karasu. As i mentioned before, If your connection to main 
Antergos repo is blocked by any means, (for example university 
firewall), the installer fails like this. I tried editting the 
antergos-mirrorlist myself, but still It's  throwing me this error.
Now, If i use a vpn (Which i really don't like this way), the installer 
successfully goes on.

**Comment by lots0logs** [comment]
Cnchi requires an unrestricted internet connection in order to complete installation. Its up to the user to ensure that they are not behind a firewall or that the proper measures have been taken to ensure communication through the firewall.
